pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -233,6 +233,9 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			printLines(out, indent, 0, describeMonopod(f, monopod.Pod)...)
 		}
 
+		bcPipelines, _ := graphview.AllImagePipelinesFromBuildConfig(g, graphview.IntSet{})
```

Comment:
> Isn't this redundant information from what we are already showing?

It wasn't before when I wrote up the bug.  There was no way to relate to the two and when you got more than one in, it was very difficult to follow.  I like the "produced by" text in @gabemontero's example.



> Is a build chain useful to me?

The chain is useful when you're trying to kick a build by rebuilding its input image.  It comes up for me when I'm trying to update base layers which contain utilities and prereqs.  I have one image with fedora plus basic tools (everything I build runs on top of this), another image with specific prereqs (think: this app needs ffmpeg), and finally my top layer.  I want to rebuild image two, not one or three.  But the build I remember is for image three.  This lets me track backwards.